match/lib/match/options.rb

```diff
@@ -117,7 +117,18 @@ def self.available_options
                                      env_name: "MATCH_SKIP_DOCS",
                                      description: "Skip generation of a README.md for the created git repository",
                                      is_string: false,
-                                     default_value: false)
+                                     default_value: false),
+        FastlaneCore::ConfigItem.new(key: :platform,
+                                     short_option: '-o',
+                                     env_name: "MATCH_PLATFORM",
+                                     description: "Set the provisioning profile's platform to work with (i.e. ios, tvos)",
+                                     is_string: true,
```

Comment:
@lacostej I tried that but I could not get it to work. Idk, if its because some parts wanted a symbol and some wanted a string.